### PR TITLE
docs(node): Unify serverless node docs

### PR DESCRIPTION
**Pull request by kamilogorek**

### CODE REVIEW

File: src/collections/_documentation/platforms/node/aws_lambda.md [hunk 1]
```diff
@@ -7,40 +7,41 @@ Create a deployment package on your local machine and install the required depen
 
 Install our Node.js SDK using `npm`:
 
-```basic
-npm install @sentry/node
+```bash
+$ npm install @sentry/node
```
Comment (rhcarvalho) [inline src/collections/_documentation/platforms/node/aws_lambda.md:11]: Need to check what syntax highlighting lib we use, but normally `$` goes with ` ```console`.
Sometimes `$` is visually good, but has the disadvantage of making it slightly harder to copy-paste.

bash:

```bash
$ npm install @sentry/node
results
```

console:

```console
$ npm install @sentry/node
results
```

```shell
npm install @sentry/node
```

^^ no `$` allows you to double click to select the whole line, copy, paste.

File: src/collections/_documentation/platforms/node/aws_lambda.md [hunk 2]
```diff
@@ -7,40 +7,41 @@ Create a deployment package on your local machine and install the required depen
...
 ```
-You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
 
-Note: You need to call both `CaptureException` and `Flush` to report errors into Sentry.
+You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
 
-Create the deployment package in .zip format, then upload it to AWS Lambda as a Lambda function. Checkout Sentry’s [aws sample apps](https://github.com/getsentry/examples/tree/master/aws-lambda/node) for detailed examples. Use [JavaScript docs](/platforms/javascript/) for more configuration options.
+Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
 
+Create the deployment package in `.zip` format, then upload it to AWS Lambda as a Lambda function. Checkout Sentry's [aws sample apps](https://github.com/getsentry/examples/tree/master/aws-lambda/node) for detailed examples. Use [JavaScript docs](/platforms/javascript/) for more configuration options.
```
Comment (rhcarvalho) [inline src/collections/_documentation/platforms/node/aws_lambda.md:47]:
```suggestion
Create the deployment package in `.zip` format, then upload it to AWS Lambda as a Lambda Function. Checkout Sentry's [aws sample apps](https://github.com/getsentry/examples/tree/master/aws-lambda/node) for detailed examples. Refer to the [JavaScript docs](/platforms/javascript/) for more configuration options.
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 3]
```diff
@@ -3,36 +3,40 @@ title: Azure Functions
...
+Install our Node.js SDK using `npm`:
+
 ```bash
-npm install @sentry/node
+$ npm install @sentry/node
 ```
 
-Set up Sentry error logging for your azure function:
-```js
+To set up Sentry error logging for a Azure Function:
```
Comment (rhcarvalho) [inline src/collections/_documentation/platforms/node/azure_functions.md:12]:
```suggestion
To set up Sentry error logging for an Azure Function:
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 4]
```diff
@@ -3,36 +3,40 @@ title: Azure Functions
...
+  };
 }
 ```
 
 You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
 
 Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
 
-Checkout Sentry’s [azure sample apps](https://github.com/getsentry/examples/tree/master/azure-functions/node) for detailed examples. For detailed configuration options, use [JavaScript docs](/platforms/javascript/).
+Checkout Sentry's [azure sample apps](https://github.com/getsentry/examples/tree/master/azure-functions/node) for detailed examples. Use [JavaScript docs](/platforms/javascript/) for more configuration options.
```
Comment (rhcarvalho) [inline src/collections/_documentation/platforms/node/azure_functions.md:42]:
```suggestion
Checkout Sentry's [Azure sample apps](https://github.com/getsentry/examples/tree/master/azure-functions/node) for detailed examples. Refer to the [JavaScript docs](/platforms/javascript/) for more configuration options.
```

File: src/collections/_documentation/platforms/node/gcp_functions.md [hunk 5]
```diff
@@ -32,8 +34,9 @@ exports.cloud_handler = (event, context) => {
   };
 };
 ```
+
 You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
 
 Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
 
-Checkout Sentry’s [gcp sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions/node) for detailed examples. For detailed configuration options, use [JavaScript docs](/platforms/javascript/).
+Checkout Sentry's [azure sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions/node) for detailed examples. Use [JavaScript docs](/platforms/javascript/) for more configuration options.
```
Comment (rhcarvalho) [inline src/collections/_documentation/platforms/node/gcp_functions.md:42]: Ooopps, gcp->azure->GCP :)

```suggestion
Checkout Sentry's [GCP sample apps](https://github.com/getsentry/examples/tree/master/gcp-cloud-functions/node) for detailed examples. Refer to the [JavaScript docs](/platforms/javascript/) for more configuration options.
```

File: src/collections/_documentation/platforms/node/aws_lambda.md (hunk 1 above)
Comment (kamilogorek) [reply src/collections/_documentation/platforms/node/aws_lambda.md:11]: True, but I wanted to follow what we already use in majority of examples (31 matches for `$ npm install`, 10 matches for `npm install`)

File: src/collections/_documentation/platforms/node/aws_lambda.md [hunk 6]
```diff
@@ -7,40 +7,41 @@ Create a deployment package on your local machine and install the required depen
...
 module.exports.hello = sentryHandler(async event => {
-  throw new Error("asd");
+  notExistFunction();
+  return event;
 });
 ```
-You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
 
-Note: You need to call both `CaptureException` and `Flush` to report errors into Sentry.
+You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/node/aws_lambda.md:43]:
```suggestion
You can obtain the DSN using your Sentry account from your organization's *Settings > Projects > Client Keys (DSN)* in the Sentry web UI.
```

File: src/collections/_documentation/platforms/node/aws_lambda.md [hunk 7]
```diff
@@ -7,40 +7,41 @@ Create a deployment package on your local machine and install the required depen
...
 
-Create the deployment package in .zip format, then upload it to AWS Lambda as a Lambda function. Checkout Sentry’s [aws sample apps](https://github.com/getsentry/examples/tree/master/aws-lambda/node) for detailed examples. Use [JavaScript docs](/platforms/javascript/) for more configuration options.
+Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/node/aws_lambda.md:45]:
```suggestion
Note: You need to call both `captureException` and `flush` for captured events to be successfully delivered to Sentry.
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 8]
```diff
@@ -3,36 +3,40 @@ title: Azure Functions
...
+  }
+
+  context.res = {
+    status: 200,
+    body: "Hello from Azure Cloud Function!"
+  };
 }
 ```
 
 You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/node/azure_functions.md:38]:
```suggestion
You can obtain the DSN using your Sentry account from your organization's *Settings > Projects > Client Keys (DSN)* in the Sentry web UI.
```

File: src/collections/_documentation/platforms/node/azure_functions.md [hunk 9]
```diff
@@ -3,36 +3,40 @@ title: Azure Functions
...
 
 Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/node/azure_functions.md:40]:
```suggestion
Note: You need to call both `captureException` and `flush` for captured events to be successfully delivered to Sentry.
```

File: src/collections/_documentation/platforms/node/gcp_functions.md [hunk 10]
```diff
@@ -32,8 +34,9 @@ exports.cloud_handler = (event, context) => {
   };
 };
 ```
+
 You can obtain the DSN using your Sentry account from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/node/gcp_functions.md:38]:
```suggestion
You can obtain the DSN using your Sentry account from your organization's *Settings > Projects > Client Keys (DSN)* in the Sentry web UI.
```

File: src/collections/_documentation/platforms/node/gcp_functions.md [hunk 11]
```diff
@@ -32,8 +34,9 @@ exports.cloud_handler = (event, context) => {
...
 
 Note: You need to call both `captureException` and `flush` in order for captured events to be successfully delivered to Sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/node/gcp_functions.md:40]:
```suggestion
Note: You need to call both `captureException` and `flush` for captured events to be successfully delivered to Sentry.
```

**Pull request merged**